### ISSUE TITLE
Better setup instructions?

### ISSUE THREAD

**Issue by dannagle** [dannagle/PiPlayer#1]
Since there is no intention of providing installers, it'd be nice if this tool provided more detailed compile instructions or a script to help set things up.

**Comment by nguyengiap1467** [comment]
I have issue when run service pi. I have some question?
1. Where are the file services located?
2. How to build of service.
3. How to run daemon service.
Thanks !

**Comment by dannagle** [comment]
I agree that setup instructions could be better. I am leaving this ticket open as a reminder to update instructions soon.
